.travis.yml

```
@@ -6,13 +6,13 @@ php:
   - 8.1
 
 node_js:
-  - "14"
+  - "18"
```

Keep support for node 16?

```suggestion
  - "16"
  - "18"
```